src/main.rs

```diff
@@ -0,0 +1,86 @@
+use actions_github::context::get_context;
+use actions_github::core::{get_input, set_output};
+use actions_github::logger;
+use anyhow::Result;
+use octocrab::Octocrab;
+
+#[tokio::main]
+async fn main() {
+    let ctx = match get_context() {
+        Ok(context) => context,
+        Err(error) => panic!("{}", error),
+    };
+    let org: String = ctx.repo.owner;
+
+    let team_name: String = get_variable("team", true);
+
+    let token: String = get_variable("ACCESS_TOKEN", true);
```

Comment:
`get_variable` function seems redundant, as these are the only usages, and it's basically
```suggestion
    let team_name: String = get_input("team").unwrap();

    let token: String = get_input("ACCESS_TOKEN").unwrap();
```